src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -893,11 +894,23 @@ public string RoleClaimType
         /// </summary>
         public IEnumerable<string> ValidAudiences { get; set; }
 
+        private string _validIssuer;
+
         /// <summary>
         /// Gets or sets a <see cref="string"/> that represents a valid issuer that will be used to check against the token's issuer.
         /// The default is <c>null</c>.
         /// </summary>
-        public string ValidIssuer { get; set; }
+        public string ValidIssuer
```

Comment:
If we can save off the utf8 bytes in a ValueSpan, then we can adjust the validation of the issuer to check for the uft8 span.

Reply:
I don't fully understand what you mean. Do you mean as internally everywhere we should use `ReadOnlySpan<byte> ValidIssuerUtf8` when we can instead of the string property?